src/api/import.js

```diff
@@ -161,7 +161,10 @@ export default async (providedOptions = {}) => {
     d('deleting old dependencies forcefully');
     await fs.remove(path.resolve(dir, 'node_modules/.bin/electron'));
     await fs.remove(path.resolve(dir, 'node_modules/.bin/electron.cmd'));
-    await fs.remove(path.resolve(dir, 'node_modules', electronName));
+
+    if (electronName !== undefined) {
```

Comment:
No prob, will change rn. We're just sticklers for explicit bools internally (been burned too many times by items that evaluate to e.g. `if (0) { ... }`)